### PR TITLE
Divided viewport

### PR DESCRIPTION
**Pull request by Ellord207**

### CODE REVIEW

**Comment by Ellord207** [comment]
There is an bug I found where, if you lock the clip and change the comparison type, the comparison does not change.

**Review by pmconne — APPROVED** [review]
I have only nitpicks, which I will spare you. Good readable example.

**Review by williamkbentley — APPROVED** [review]

**Review by Josh-Schifter — CHANGES_REQUESTED** [review]
The UI is inconsistent with the other samples.

- Looks like you are using the wrong selector component
- The spacing between the labels and the other components is wrong.is wrong.

File: src/frontend-samples/swiping-viewport/Divider.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      width = parseInt(widthStr, 10);
+    return width;
+  }
+  private limitToBounds(n: number): number {
+    n = Math.min(n, this.props.bounds.right - (this._elem!.clientWidth + this._buffer));
+    n = Math.max(n, this.props.bounds.left + this._buffer);
+    return n;
+  }
+
+  constructor(props: DividerComponentProps, context: any) {
```
Comment (Josh-Schifter) [inline src/frontend-samples/swiping-viewport/Divider.tsx:56]: Remove the "context" argument.  The code editor in the showcase reports this as a "problem" at runtime.

File: src/frontend-samples/swiping-viewport/SwipingComparisonApp.tsx [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+  GraphicBranch,
+  IModelApp,
+  IModelConnection,
+  RenderClipVolume,
+  SceneContext,
+  ScreenViewport,
+  TiledGraphicsProvider,
+  TileTreeReference,
+  Viewport,
+} from "@bentley/imodeljs-frontend";
```
Comment (Josh-Schifter) [inline src/frontend-samples/swiping-viewport/SwipingComparisonApp.tsx:35]: I think we should collapse each of these import statements into a single line.  At present, the imports take up an entire screen in the code editor at runtime.

**Review by Josh-Schifter — CHANGES_REQUESTED** [review]

**Review by Josh-Schifter — APPROVED** [review]

**Pull request merged**